TaskManagementSystem/Services/TasksService.cs

```diff
@@ -56,27 +61,35 @@ public async Task<TaskResponseDto> GetTaskByIdAsync(int taskId)
         var taskEntity = await _dbContext.Tasks.FindAsync(taskId);
 
         if (taskEntity is null)
-            throw new NotFoundException(ValidationMessages.TaskDoesNotExist);
+            throw new NotFoundException(ErrorMessageConstants.TaskDoesNotExist);
 
         return taskEntity.ToOutDto();
     }
 
-    public async Task<TaskResponseDto?> UpdateTaskAsync(int taskId, TaskRequestDto taskDto)
+    public async Task<TaskResponseDto?> UpdateTaskAsync(int taskId, UpdateTaskRequestDto taskDto)
```

Comment:
why taskId is duplicated outside taskDto?

Reply:
The taskId is part of the URL while taskDto is body parameter, maybe we can introduce single DTO which unites them